package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "node.mentoring.program.2018",
+  "version": "1.0.0",
+  "description": "NodeJS Mentoring program",
+  "main": "index.js",
+  "repository": "https://github.com/kuzanatoliy/node.mentoring.program.2018.git",
+  "author": "Anatoli Kuzmiankou <Anatoli_Kuzmiankou@epam.com>",
+  "license": "MIT",
+  "scripts": {
+    "lint": "eslint . --ext .js",
+    "dev": "nodemon index.js",
+    "start": "node index.js",
```

Comment:
Following the requirement of the task, start script should compile app.js using babel and run it in nodemon. To make use of nodemon and babel, you can run this command: **nodemon app.js --exec babel-node**